src/database/mixxxdb.cpp

```diff
@@ -1,17 +1,15 @@
 #include "database/mixxxdb.h"
 
 #include "database/schemamanager.h"
-
 #include "util/assert.h"
 #include "util/logger.h"
 
-
 // The schema XML is baked into the binary via Qt resources.
 //static
 const QString MixxxDb::kDefaultSchemaFile(":/schema.xml");
 
 //static
-const int MixxxDb::kRequiredSchemaVersion = 33;
+const int MixxxDb::kRequiredSchemaVersion = 35;
```

Comment:
what happened to 34?

Reply:
Sorry, got lost between merge conflict resolutions. I have too many open PRs. One reason why I now have to suspend any Mixxx development, and the lack of time.

Reply:
35 is correct, I just got confused by our remark. The missing indexes are added in a separate step before the actual schema migration.